waspc/test/Analyzer/Parser/ParseErrorTest.hs

```diff
@@ -1,45 +1,38 @@
 module Analyzer.Parser.ParseErrorTest where
 
+import Analyzer.TestUtil (ctx, pos)
 import Test.Tasty.Hspec
 import Wasp.Analyzer.Parser.ParseError
 import Wasp.Analyzer.Parser.Token
 
 spec_ParseErrorTest :: Spec
 spec_ParseErrorTest = do
   describe "Analyzer.Parser.ParseError" $ do
-    let unexpectedCharError = UnexpectedChar '!' (SourcePosition 2 42)
-        unexpectedTokenErrorNoSuggestions =
-          UnexpectedToken (Token TLCurly (SourcePosition 2 3) "{") []
-        unexpectedTokenErrorWithSuggestions =
-          UnexpectedToken
-            (Token TRCurly (SourcePosition 100 18) "}")
-            ["<identifier>", ","]
-        quoterDifferentTagsError =
-          QuoterDifferentTags
-            ("foo", SourcePosition 1 5)
-            ("bar", SourcePosition 1 20)
-    describe "getErrorMessage returns human readable error message" $ do
+    describe "getErrorMessageAndCtx returns human readable error message and correct position" $ do
```

Comment:
probably some a/the missing here, grammarly this

Reply:
Fixed!